### ISSUE TITLE
remove duplicates from problem listing

### ISSUE THREAD

**Issue by jakoch** [composer/composer#3768]
Steps to reproduce the issue:

`composer.json`

    {
        "require": {
            "php": ">=9.1"
        }
    }

`composer install`
```
Loading composer repositories with package information
Installing dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - The requested package php could not be found in any version, there may be a typo in the package name.
  Problem 2
    - The requested package php could not be found in any version, there may be a typo in the package name.

Potential causes:
 - A typo in the package name
 - The package is not available in a stable-enough version according to your minimum-stability setting
   see <https://groups.google.com/d/topic/composer-dev/_g3ASeIFlrc/discussion> for more details.

Read <http://getcomposer.org/doc/articles/troubleshooting.md> for further common problems.
```

There are two Problems reported and both messages are the same.
I suggest to remove the duplicates, so that all reported problems are unique.

**Comment by sstok** [comment]
:+1:

**Comment by xabbuh** [comment]
@jakoch Updating Composer will probably solve your issue. For me, using the latest version produces this output:

```
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - This package requires php >=9.1 but your PHP version does not satisfy that requirement.
```

I think this was solved with 3ba581f0dfa9560e04dcfa0a90ff60c26f20edb3.

**Comment by jakoch** [comment]
Yes, solved. Closing.
